wireapp/wire-android
# feat: Force to update (AR-2714)

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-2714" title="AR-2714" target="_blank"><img alt="Story" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10815?size=medium" />AR-2714</a>  Forcing updates
  </td></table>
  <br />
 


















# What's new in this PR?

Force user to update the app if it's required.

Needs releases with:

- Merge after [kalium PR](https://github.com/wireapp/kalium/pull/1169)

#### How to Test


Tricky thing: to test it we need to have an app in the market AND install the app from the market. 
AND add the version that was installed from the market into "blacklist" . 

If all of it is true - user should see the dialog that blocks user from using the app and informs that app has to be updated.


### Attachments (Optional)

_Attachments like images, videos, etc. (drag and drop in the text box)_
